mlrun/config.py

```diff
@@ -79,6 +79,9 @@
         'data_volume': '',
         'real_path': '',
         'db_type': 'sqldb',
+        # the minimum interval (in minutes) that will be allowed between two scheduled jobs - e.g. a job wouldn't be
+        # allowed to be scheduled to run more then 2 times in X minutes
+        'minimum_interval_between_scheduled_jobs': '10',
```

Comment:
Suggest:
1. scheduling: {
    'min_allowed_interval_minutes': '10'
}


2. maybe human friendly instead of "hardcoding" the resolution to minutes